src/sentry/services/hybrid_cloud/region.py

```diff
@@ -4,12 +4,12 @@
 from dataclasses import dataclass
 from typing import TYPE_CHECKING
 
-from sentry.db.models import BaseManager
 from sentry.services.hybrid_cloud import ArgumentDict
 from sentry.services.hybrid_cloud.rpc import RpcServiceUnimplementedException
 from sentry.types.region import Region, get_region_by_name
 
 if TYPE_CHECKING:
+    from sentry.db.models import BaseManager
```

Comment:
This is the only non user-service import change. It's also necessary so we're not initializing the models module while setting up the region silo.